manifests/manifest-preprod.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- name: response-operations-social-ui
+  disk_quota: 1G
+  instances: 1
+  memory: 512M
+  stack: cflinuxfs2
+  timeout: 180
+  services:
+  - res-ops-redis
```

Comment:
should this be `ras-ops-redis`?

Reply:
It's possibly a type in Prod CF but it is the correct string from our point of view.

Reply:
I'll raise a tech debt to at least flag it.